lms/djangoapps/discussion/templates/discussion/edx_ace/response_notification/email/body.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% load static %}
+
+{% block preview_text %}
+    {% blocktrans trimmed %}
+        Hi {{ thread_username }}
+    {% endblocktrans %}
+{% endblock %}
+
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            {% blocktrans trimmed %}
+                <h1>
+                    Hi {{ thread_username }},
+                </h1>
+
+                <p>
+                    {{ comment_username }} made the following reply to {{ thread_title }} at {{ comment_created_at }}.
+                </p>
+
+                <p>
+                    {{ comment_body }}
+                </p>
+
+                <p>
+                    <a href="{{ post_link }}"> View the discussion.</a>
```

Comment:
Should we have an unsubscribe link, too, or did we decide against that?

Reply:
I plucked that out - I think sailthru does that for us.

Reply:
Can we hook up unsubscribing to emails through sailthru with unsubscribing/unfollowing(?) threads in the platform?